packages/components/src/tag/docs/TagGroup.stories.tsx

```diff
@@ -0,0 +1,300 @@
+                <TagList>
+                    <Tag id="1" textValue="Developer"><SparklesIcon /><Text>Developer</Text></Tag>
+                    <Tag id="2" textValue="Designer">
+                        <Text>Designer</Text>
+                        <IconList>
+                            <SparklesIcon /><SparklesIcon /><SparklesIcon />
+                        </IconList>
+                    </Tag>
+                </TagList>
+            </TagGroup>
+            <TagGroup {...props} aria-label="tag-group" size="lg">
+                <TagList>
+                    <Tag id="1" textValue="Developer"><SparklesIcon /><Text>Developer</Text></Tag>
+                    <Tag id="2" textValue="Designer">
+                        <Text>Designer</Text>
+                        <IconList>
+                            <SparklesIcon /><SparklesIcon /><SparklesIcon />
+                        </IconList>
+                    </Tag>
+                </TagList>
+            </TagGroup>
+        </Stack>
+    ),
+    args: {
+        onRemove: (ids: Selection) => {
+            alert(`Remove: ${[...ids]}`);
+        }
+    }
+} satisfies Story;
+
+
+/**
+ * Tags can also have a count using the `Badge` component.
+ */
+export const Count = {
+    render: props => (
+        <Stack>
+            <TagGroup {...props} aria-label="tag-group" size="md">
+                <TagList>
+                    <Tag id="1" textValue="Designer">
+                        <Text>Designer</Text>
+                        <Badge>12</Badge>
+                    </Tag>
+                    <Tag id="2" textValue="Developer">
+                        <Text>Developer</Text>
+                        <Badge variant="secondary">100</Badge>
+                    </Tag>
+                </TagList>
+            </TagGroup>
+            <TagGroup {...props} aria-label="tag-group" size="lg">
+                <TagList>
+                    <Tag id="1" textValue="Designer">
+                        <Badge>12</Badge>
+                        <Text>Designer</Text>
+                    </Tag>
+                    <Tag id="2" textValue="Developer">
+                        <Text>Developer</Text>
+                        <Badge variant="secondary">100</Badge>
+                    </Tag>
+                </TagList>
+            </TagGroup>
+        </Stack>
+    )
+} satisfies Story;
+
+/**
+ * A TagGroup can have a description to provide more information to the user.
+ */
+export const Description = {
+    render: props => (
+        <TagGroup {...props}>
+            <Label>List of Tags</Label>
+            <TagList>
+                <Tag id="1" textValue="Tag 1"><SparklesIcon /><Text>Tag 1</Text></Tag>
+                <Tag id="2" textValue="Tag 2"><SparklesIcon /><Text>Tag 2</Text></Tag>
+                <Tag id="3" textValue="Tag 3"><SparklesIcon /><Text>Tag 3</Text></Tag>
+            </TagList>
+            <HelperMessage>Select one to continue</HelperMessage>
```

Comment:
replace all select tags to continue helper message with something else. It's awkward in this context since tags aren't selectionable